File: modul2/Python/PRAK205-2310817210023-RifkyPutraMahardika.py
import math

A, B = map(float, input().split())
C = math.sqrt(pow(B, 2) - pow(A, 2))
Keliling = A+B+C
Luas = (C*A)/2
print(f"Alas = {C:.0f} cm")
print(f"Tinggi = {A:.0f} cm")
print(f"Keliling = {Keliling:.0f} cm")
print(f"Luas = {Luas:.0f} cm^2")

File: modul2/C/PRAK205-2310817210023-RifkyPutraMahardika.c
#include<stdio.h>
#include<math.h>
int main(){
float A, B;
scanf("%f %f", &A, &B);

float C = sqrt(pow(B, 2) - pow(A, 2));
float Keliling = A+B+C;
float Luas = (C*A)/2;
printf("Alas = %.f cm\n", C);
printf("Tinggi = %.f cm\n", A);
printf("Keliling = %.f cm\n", Keliling);
printf("Luas = %.f cm^2\n", Luas);
}